client/src/app/+about/about.component.html

```diff
@@ -10,4 +10,4 @@
     </ul>
     <p>הנתונים שמוצגים בפנסיה פתוחה לקוחים מהדו”חות המפורסמים ע"י הגופים המנהלים של קרנות הפנסיה. דו”חות אלו המציגים פירוט ברמת הנכס הבודד של הנכסים המוחזקים בכל קרן. הדו”חות מתפרסמים בסוף כל רבעון, ופנסיה פתוחה מעדכנת את מאגר הנתונים שלה לאחר מכן. לפנסיה פתוחה אין כל אחריות לנכונות ו/או לשלמות הנתונים המוצגים.&nbsp;</p>
     <p>בשלב זה, פנסיה פתוחה מציגה רק את הנתונים של חברת מגדל. בעתיד הקרוב, פנסיה פתוחה תקיף את כל שוק הפנסיה: קרנות פנסיה, קופות גמל וביטוחי מנהלים.</p>
-      
\ No newline at end of file
+  
```

Comment:
this is good or another more backspace? i have hard ocd, so i wont sleep at night till it be alright.